tensorboard/plugins/debugger_v2/debug_data_multiplexer.py

```diff
@@ -136,56 +115,73 @@ def __init__(self, logdir):
         self._logdir = logdir
         self._reader = None
         self._reader_lock = threading.Lock()
-        self._createOrReloadReader()
+        self._reload_needed_event = None
+        # Create the reader for the tfdbg2 data in the lodir as soon as
+        # the backend of the debugger-v2 plugin is created, so it doesn't need
+        # to wait for the first request from the FE to start loading data.
+        self._tryCreateLoader()
 
-    def _createOrReloadReader(self):
+    def _tryCreateLoader(self):
```

Comment:
It seems clearer and more consistent to call this `_tryCreateReader`?  That's the term used in other places, and in the implementation, since it creates a `DebugDataReader` stored in `self._reader`.

Reply:
Done.